lib/measure_repository_service_test_kit/measure_group.rb

```diff
@@ -5,7 +5,7 @@
 module MeasureRepositoryServiceTestKit
   # tests for read by ID and search for Measure service
   class MeasureGroup < Inferno::TestGroup
-    title 'Measure Repository Service Measure Group'
+    title 'Measure Group'
```

Comment:
The Measure Group and Library Group will both capture read by id and search, so maybe we should consider changing them to "Measure Read by Id and Search"/"Library Read by Id and Search" (or something simpler than that like "Measure Read/Search"). I agree that adding the package operation to the Measure group will make it too big (and it's testing a separate operation so it should prob be it's own test group anyway)

Reply:
I'm not attached to the current test group names so feel free to make those changes in this PR

Reply:
@elsaperelli I def agree and think the title could benefit from more clarity. I think it's within scope for my task since I already changed it once :)